### PR TITLE
GH-6798 added the doc for the bulk reactions endpoint

### PR DESCRIPTION
**Pull request by pradeepmurugesan**
Added the api doc for the new endpoint for bulk selecting reactions to posts.
Ticket - https://github.com/mattermost/mattermost-server/issues/6798

### CODE REVIEW

File: v4/source/reactions.yaml [hunk 1]
```diff
@@ -89,3 +89,33 @@
...
+        - reactions
+      summary: Bulk get the reaction for posts
+      description: |
+        Get a list of reactions made by all users to a given post.
+        ##### Permissions
+        Must have `read_channel` permission for the channel the post is in.
+      parameters:
+        - in: body
+          name: postIds
+          description: Array of postIds
```
Comment (jwilander) [inline v4/source/reactions.yaml]:
```suggestion
           description: Array of post IDs
```

File: v4/source/reactions.yaml [hunk 2]
```diff
@@ -89,3 +89,33 @@
...
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        '200':
+          description: Reaction deletion successful
```
Comment (jwilander) [inline v4/source/reactions.yaml]:
```suggestion
           description: Reactions retrieval successful
```

**Comment by pradeepmurugesan** [comment]
Thanks for reviewing @jwilander . Fixed the ~review comments~ copy paste error :-)

File: v4/source/reactions.yaml [hunk 3]
```diff
@@ -89,3 +89,33 @@
...
+
+  '/posts/ids/reactions':
+    post:
+      tags:
+        - reactions
+      summary: Bulk get the reaction for posts
+      description: |
+        Get a list of reactions made by all users to a given post.
+        ##### Permissions
+        Must have `read_channel` permission for the channel the post is in.
```
Comment (jwilander) [inline v4/source/reactions.yaml]: Can you add a minimum server version of 5.8 here? You can see how other endpoints do it as an example

**Comment by pradeepmurugesan** [comment]
@jwilander added the same 👍 .. Kindly review

**Comment by jwilander** [comment]
Thanks @pradeepmurugesan !

**Pull request merged**